python/cudf/cudf/core/dataframe.py

```diff
@@ -7032,6 +7032,8 @@ def unstack(self, level=-1, fill_value=None):
             self, level=level, fill_value=fill_value
         )
 
+    _accessors = set()
```

Comment:
Since this attribute is common to DataFrame, Series and Index, we can define it once, in the base class `Frame`

Reply:
Don't we also want to separate `DataFrame._accessors` from `Series._accessors` and `Index._accessors`? If defined in `Frame` they become the same set space.

Reply:
You're right - I missed that :)